qiskit_experiments/library/tomography/fitters/cvxpy_utils.py

```diff
@@ -64,7 +64,9 @@ def decorated_func(*args, **kwargs):
     return decorated_func
 
 
-def solve_iteratively(problem: Problem, initial_iters: int, scale: int = 2, **solve_kwargs) -> None:
+def solve_iteratively(
+    problem: Problem, initial_iters: int, scale: int = 2, solver: str = "SCS", **solve_kwargs
```

Comment:
Do you need a release note to document the new argument being added to this function?

Reply:
These util functions are currently only used internally in tomography experiments and aren't exposed in the API, so I didn't feel like a release note is needed.

Reply:
Sure, makes sense. I wasn't sure whether it was public or not because it had a detailed docstring :)